integration/localnet/Makefile

```diff
@@ -55,9 +55,10 @@ else
 endif
 
 # Creates a light version of the localnet with just 1 instance for each node type
+# (Except for consensus and execution nodes, which are set to 2)
 .PHONY: init-light
 init-light:
-	$(MAKE) -e COLLECTION=1 CONSENSUS=2 EXECUTION=1 VERIFICATION=1 ACCESS=1 NCLUSTERS=1 init
+	$(MAKE) -e COLLECTION=1 CONSENSUS=2 EXECUTION=2 VERIFICATION=1 ACCESS=1 NCLUSTERS=1 init
```

Comment:
why are these numbers hardcoded?  should they be the same as the constants defined at the top of the file?

Reply:
these are overrides for the pre-defined ones to bootstrap the smalelst-viable-cluster.  We could reuse the default values from the top in cases where they match, but we do not want them to accidentally change in the future.

I can factor out the "2" constant, but it looks like it would make code less readable.